Automattic/newspack-plugin
# fix(ras): update reset password email template heading and text

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://app.asana.com/0/1207817176293825/1208629737155851/f

This PR updates the password reset heading, text, and button to be more general for both new users setting up a password for the first time, as well as for existing users who already have a password set:

![Screenshot 2024-10-27 at 11 37 13](https://github.com/user-attachments/assets/68d5f93c-c7a3-464a-b144-de2116a92759)

### How to test the changes in this Pull Request:

1. As admin, go to Newspack > Engagement > Show Advanced Settings > Transactional Email Content
2. Reset the `Set a New Password` template
3. Edit the template and confirm the text matches the screenshot above
4. Send a test email and confirm the text matches again

### Other information:
